lib/srv/desktop/rdp/rdpclient/src/cliprdr.rs

```diff
@@ -990,7 +942,7 @@ mod tests {
         // First response - our client capabilities:
         let mut payload = Cursor::new(responses[0].clone());
         let _pdu_header = vchan::ChannelPDUHeader::decode(&mut payload).unwrap();
-        let header = ClipboardPDUHeader::decode(&mut payload).unwrap();
+        let header = <ClipboardPDUHeader>::decode(&mut payload).unwrap();
```

Comment:
What's this about?

Reply:
I don't know. How is that syntax even valid?

Reply:
It's called Turbofish. But I don't think it's necessary here.